.github/workflows/cacerts-apply-patch.sh

```diff
@@ -2,14 +2,13 @@
 
 set -euo pipefail
 
-UPDATE_NEEDED=1
-
 . .github/workflows/common.sh
 
+BRANCH_NAME="cacerts-${VERSION_NEW}-${TARGET}"
+
 prepare_git_repo
 
-if ! checkout_branches "${VERSION_NEW}-${TARGET}"; then
-  UPDATE_NEEDED=0
+if ! checkout_branches "${BRANCH_NAME}"; then
   exit 0
 fi
```

Comment:
We set `UPDATE_NEEDED` in order not to push unnecessary commits to existing open PRs that were created for the same branch.
We have a history of having troubles with noisy commits for existing open PRs, when we tried to touch `UPDATE_NEEDED` in that code.
If we would want to remove that part like above, I would recommend testing the specific scenarios using this PR.

Reply:
UPDATE_NEEDED is set to zero, then we exit the script. The whole point of this variable is to be a substitution at the bottom of the script where we do `echo ::set_output` stuff. I'll rerun the workflows on my fork - there are already those branches, so next workflow run should just stop here.

Reply:
As a matter of fact, the [action for kernel updates run on my fork 4 hours ago](https://github.com/krnowak/coreos-overlay/actions/runs/3248096616) and all the jobs stopped at this checkout_checkout branches function with the message "Target branch already exists. exit." and no PR was created/updated. I've started a run for ca-certificates and guess what - got another set of 6 PRs because [ca-certificates 3.84 got released yesterday](https://firefox-source-docs.mozilla.org/security/nss/nss_3_84.html#mozilla-projects-nss-nss-3-84-release-notes). :) But reruninng it again yielded no new PRs nor any PR updates.